### PR TITLE
route utility added

### PR DESCRIPTION
**Pull request by ukirderohit**

### CODE REVIEW

File: packages/im-components/src/organisms/filmList/FilmList.js [hunk 1]
```diff
@@ -44,16 +49,24 @@ const renderMovie = (
...
 ) => {
   const filmId = FilmReader.id(filmDetails);
   const filmTitle = FilmReader.title(filmDetails);
   const filmRating = FilmReader.rating(filmDetails);
   const filmGenre = FilmReader.genre(filmDetails);
   const filmImgSrc = FilmReader.thumbnail(filmDetails);
 
+  let filmDetailsRoute;
+  if (application === APPS.OTT) {
+    filmDetailsRoute = getRoute(APPS.OTT, VIEWS.FILMDETAILS, { filmId });
```
Comment (pranay2401) [inline packages/im-components/src/organisms/filmList/FilmList.js:62]: why the if check? Pass application directly to getRoute.

File: packages/im-components/src/organisms/filmList/FilmList.js [hunk 2]
```diff
@@ -97,6 +110,14 @@ const FilmList = (props) => {
 
   const filmListContainer = getFilmListClassName(isFeatured);
 
+  let movieListRoute;
+  if (application === APPS.OTT) {
+    movieListRoute = getRoute(APPS.OTT, VIEWS.MOVIELIST, { listKey });
+  }
```
Comment (pranay2401) [inline packages/im-components/src/organisms/filmList/FilmList.js]: same. No if check needed

**Review by pranay2401 — CHANGES_REQUESTED** [review]

File: packages/im-components/src/organisms/filmList/FilmList.js (hunk 1 above)
Comment (ukirderohit) [reply packages/im-components/src/organisms/filmList/FilmList.js:62]: This is a generic utility.. can be used for both studio and ott routes

File: packages/im-components/src/organisms/filmList/FilmList.js (hunk 2 above)
Comment (ukirderohit) [reply packages/im-components/src/organisms/filmList/FilmList.js]: same

**Pull request merged**